### DISCUSSION TITLE
Can't save file, no code completion

### DISCUSSION THREAD

**Discussion by NikRetaNCAM** [HaxeFlixel/flixel#2661] [Q&A]
I use VS Code, and usually it works just fine. Right now it decided to go crazy. It keeps showing me as if I didn't save the file, yet I did. That's being confirmed by the fact that when I try to restart the VS code or close it it tells me I have stuff unsaved, more exact screenshot:
![image](https://user-images.githubusercontent.com/60806448/199752311-3d5b175c-43c7-48b9-8442-5ccf2da3e7bb.png)
add up to this that when I type something in it does not show code completion (for example I type "Flx" when declaring a variable and it does not show me the possible variants of what i'm gonna type)

BEHAVIOR I WANT TO SEE: Visual Studio Code saving my code, providing me with code completion
BEHAVIOR I SEE: opposite of what I want to see

If that has happened to anyone else, why is it happening and how to fix it?

**Comment by NikRetaNCAM** [comment]
That probably isnt exactly haxeflixel question, sorry.

**Comment by Geokureli** [comment]
revert back to a previous vscode version
https://code.visualstudio.com/updates/v1_72
